working-groups/performance-testing/README.md

```diff
@@ -0,0 +1,92 @@
+# HPSF Benchmarking Working Group
+
+Benchmarking WG aims to bring together projects and organizations 
+to create an automated way to 
+continuously test the performance of evolving HPC software stacks, 
+ensuring that the components work and perform well.
+Functionality testing and performance regression testing are both in scope.
+Parts of the software stack we would like to test:
+* Compilers
+* Runtimes and programming models
+* MPI and other communication libraries
+* Math libraries
+* HPC applications
+  
+
+## Leads
+
+* Olga Pearce ([@pearce8](https://github.com/pearce8)])
+* ???
+
+
+## Meetings
+
+* The working group meets every **TBD** weeks. [meeting notes](https://docs.google.com/document/d/13edPtYo5hPpbFg7PO71v8RMGJByJRb1fUY1JPuwshkA/edit?usp=sharing)
+* Zoom coordinates are **TBD**.
+* To get involved, join the [Slack channel](https://hpsf.slack.com/archives/C08NKKL97NZ)
+
+
+## Motivation for launch
+
+* There is not a group that is consolidating and maintaining open source benchmarks
+* Each datacenter is for themselves
+* Testing/benchmarking of the HPC software stack is intermittent at best
+* We want to figure out, as a working group, how we can pull together resources to support and maintain continuous testing
```

Comment:
One issue is that benchmark are inherently something a bit aside from the regular development cycle for library/application teams and once the results are published for a particular machine we might not feel the need to repeat the experiment. On the other hand if the benchmark can be leveraged to observe performance improvement/regression for libraries that gives it more chance to run on a regular basis.